### ISSUE TITLE
reverse

### ISSUE THREAD

**Issue by najklez** [claytonjn/hass-circadian_lighting#167]
Hello
Is there a way i can use it backwords? like if there is a lot of sun outside, the light is nerly off. Like if the lightbulb act like a sun..?

**Comment by claytonjn** [comment]
No, this is not currently possible. I will leave this open for now to allow other users to comment if they would find it useful, but this has never been requested before so I likely won't implement this option. Can you explain why you want lights to behave this way so I can better understand your use-case?

**Comment by najklez** [comment]
Ooh... I see. I would use it for my mancave. Don't need light when sun is up but as soon it will start to get darker I would like my lightbulb, ledlist to increase the light. Skickat från min Galaxy
-------- Originalmeddelande --------Från: Clayton Nummer ***@***.***> Datum: 2021-09-14  16:59  (GMT+01:00) Till: claytonjn/hass-circadian_lighting ***@***.***> Kopia: najklez ***@***.***>, Author ***@***.***> Ämne: Re: [claytonjn/hass-circadian_lighting] reverse (#167) 
No, this is not currently possible. I will leave this open for now to allow other users to comment if they would find it useful, but this has never been requested before so I likely won't implement this option. Can you explain why you want lights to behave this way so I can better understand your use-case?

**Comment by icemanch** [comment]
> Ooh... I see. I would use it for my mancave. Don't need light when sun is up but as soon it will start to get darker I would like my lightbulb, ledlist to increase the light. Skickat från min Galaxy -------- Originalmeddelande --------Från: Clayton Nummer ***@***.***> Datum: 2021-09-14 16:59 (GMT+01:00) Till: claytonjn/hass-circadian_lighting ***@***.***> Kopia: najklez ***@***.***>, Author ***@***.***> Ämne: Re: [claytonjn/hass-circadian_lighting] reverse (#167) No, this is not currently possible. I will leave this open for now to allow other users to comment if they would find it useful, but this has never been requested before so I likely won't implement this option. Can you explain why you want lights to behave this way so I can better understand your use-case? —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

Can't this be done by setting the sunset time and sunrise time to where you want/need it?

**Comment by jferran** [comment]
I think we want the same :)
The point would be having „natural light“ in all rooms whole day, but depending on which side of the house the windows are… the timetables can be different to switch on/off the lights, or increase the power.

I considered using some automation, but think I‘d have to a customise it for each lamp, and change it during the different seasons…
